src/components/TransactionSigner.js

```diff
@@ -51,6 +53,20 @@ class TransactionSigner extends React.Component {
           href={txPostLink(result.xdr)}>Submit this transaction to the network</a>;
       }
 
+      let xdrNodes, txDetails;
+      try {
+        xdrNodes = extrapolateFromXdr(result.xdr, 'TransactionEnvelope');
+        let treeView = <TreeView className="TransactionSigner__details__tree" nodes={extrapolateFromXdr(result.xdr, 'TransactionEnvelope')} />
+        txDetails = <div className="so-back TransactionSigner__details">
+          <div className="so-chunk">
+            <p className="TransactionSigner__details__title">Transaction result details</p>
+            {treeView}
+          </div>
+        </div>
+      } catch (e) {
+        // Likely due to invalid secret key. Error messaging is handled in other sections
```

Comment:
Let's try to avoid this if we can.  Having a blank, silent catch statement can come back to bite you in the ass.  At the very least, you should specifically target an error that you expect to occur and only go silent if the thrown error matches it.

In this case, I actually expect the error to be related to decoding XDR (and not involving a secret key as mentioned in the comment), and so you should re-throw any errors that don't match that.